node/cn/tracers/internal/tracers/call_revert_tracer.js

```diff
@@ -0,0 +1,206 @@
+// Copyright 2018 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+// revertTracer returns the string of REVERT.
+// If not reverted, returns an empty string "".
+{
+    revertString: "",
+    callerAddress:"",
+    revertedContract: "",
+    callstack: [{}],
+
+    toAscii: function(hex) {
+        var str = "";
+        var i = 0, l = hex.length;
+        if (hex.substring(0, 2) === "0x") {
+            i = 2;
+        }
+        for (; i < l; i+=2) {
+            var code = parseInt(hex.substr(i, 2), 16);
+            str += String.fromCharCode(code);
+        }
+
+        return str;
+    },
+
+    // step is invoked for every opcode that the VM executes.
+    step: function(log, db) {
+        // Capture any errors immediately
+        var error = log.getError();
+        if (error !== undefined) {
+            this.fault(log, db);
+            return;
+        }
+        // We only care about system opcodes, faster if we pre-check once
+        var syscall = (log.op.toNumber() & 0xf0) == 0xf0;
+        if (syscall) {
+            var op = log.op.toString();
+        }
+        // If a new contract is being created, add to the call stack
+        if (syscall && op == 'CREATE') {
```

Comment:
Have you considered this PR related to `CREATE2` as well?
https://github.com/ethereum/go-ethereum/pull/18318

Reply:
Oh, good point! `CREATE2` should be applied to both callTracer and this.

I think this can be done another PR. :) I will add it to my TODO list.